cmd/kubeadm/app/cmd/init.go

```diff
@@ -19,27 +19,26 @@ package cmd
 import (
 	"fmt"
 	"io"
-	"io/ioutil"
 	"path"
 	"strconv"
 	"text/template"
 
 	"github.com/renstrom/dedent"
 	"github.com/spf13/cobra"
 
-	"k8s.io/apimachinery/pkg/runtime"
 	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
 	kubeadmapiext "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1alpha1"
 	"k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/validation"
+	certphase "k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases"
```

Comment:
Is used `certphase` because it is in line with other imports e.g. `addonsphase`, `apiconfigphase`, `kubeconfigphase`, `tokenphase`.

Let me know if you want to change only `certphase`, all the imports or none of them.